web/core/src/transformers.ts

```diff
@@ -1,11 +1,30 @@
-import { getPokerStepsWithEndStates } from "./transformers/repeatedPokerTransformer";
-import { GameStep } from "./types";
+import {
+  getPokerStepLabel,
+  getPokerStepsWithEndStates,
+} from "./transformers/repeatedPokerTransformer";
+import { GameStep, PokerGameStep } from "./types";
 
-export const processEpisodeData =  (steps: any[], stateHistory: any[], gameName: string): GameStep[] => {
-    switch(gameName) {
-        case "repeated_poker":
-        return getPokerStepsWithEndStates(steps, stateHistory);
-        default:
-            return steps;
-    }
-}
\ No newline at end of file
+export const processEpisodeData = (
+  steps: any[],
+  stateHistory: any[],
+  gameName: string,
+): GameStep[] => {
+  switch (gameName) {
+    case "repeated_poker":
+      return getPokerStepsWithEndStates(steps, stateHistory);
+    default:
+      return steps;
+  }
+};
+
+export const getGameStepLabel = (
+  gameStep: GameStep,
+  gameName: string,
+): string => {
+  switch (gameName) {
+    case "repeated_poker":
+      return getPokerStepLabel(gameStep as PokerGameStep);
+    default:
+      return "";
+  }
+};
```

Comment:
@develra I was imagining a very similar pattern for `getGameStepDescription` which would return the thoughts or the end-of-hand state